### PR TITLE
#146764007 Micro Python Health Check

### PR DESCRIPTION
**Pull request by Stephen-Njoroge**
#### What does this PR do?
Remove unneccessary comments on files
#### Description of Task to be completed?
Remove unneccessary comments on files
#### How should this be manually tested?
N/a
#### Any background context you want to provide?
N/a
#### What are the relevant pivotal tracker stories?
#146764007

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
     version='0.1',
     description='A library to do rpc health checks on Python Services',
     author='Andela',
     author_email='devops@andela.com',
-    url='https://github.com/andela/micro-py-health-check/tree/ft-create-micro-health-check-python-package',
-    download_url='https://github.com/andela/micro-py-health-check/tree/ft-create-micro-health-check-python-package/archive/0.1.tar.gz',
-    # I'll explain this in a second
-    keywords=['testing', 'health checks', 'services', 'gRPC'], # arbitrary keywords
+    url='https://github.com/andela/micro-py-health-check',
+    download_url='https://github.com/andela/micro-py-health-check/archive/0.1.tar.gz',
```
Comment (cyberomin) [inline setup.py]: You used `0.1.tar.gz`, will this ever change?

Comment (Stephen-Njoroge) [reply setup.py]: @cyberomin If it changes we will update it accordingly... though for consistency we install it using *branch.zip

Comment (tundeaoni) [reply setup.py]: shouldn't this be an actual release tag?

Comment (Stephen-Njoroge) [reply setup.py]: Yes, 0.1 is our current release tag, all the same we are not using the tag during installation, we use the brachurl.zip e.g `https://github.com/andela/micro-py-health-check/archive/develop.zip`

Comment (Stephen-Njoroge) [reply setup.py]: so that when we release an update, users will not be forced to update other services using the package...

**Pull request merged**